tests/foreman/cli/test_repository.py

```diff
@@ -228,8 +229,14 @@ def test_positive_create_with_default_download_policy(self):
 
         @Assert: YUM repository with a default download policy
         """
+        default_dl_policy = entities.Setting().search(
```

Comment:
Another example that happened some time ago - we got few builds with broken hammer, and that's why all our cli tests were failing. It wouldn't be better if alongside with cli, api tests were failing too.